packages/vat-data/src/types.d.ts

```diff
@@ -23,24 +22,44 @@ type ActualBehavior<B> = {
   [Facet in keyof B]: FunctionsMinusContext<B[Facet]>;
 };
 
-type KindContext<S, B> = { state: S; facets: ActualBehavior<B> };
+type MultiKindContext<S, B> = { state: S; facets: ActualBehavior<B> };
 
-interface KindDefiner {
-  <P, S, B>(
+declare const kindHandleMarker: unique symbol;
+export type DurableKindHandle = { readonly [kindHandleMarker]: string };
```

Comment:
Since the handle is a Far object, can we just make this a class with private?

```suggestion
declare class DefineKindHandleClass {
    private name: string;
}
export type DefineKindHandle = DefineKindHandleClass;
```

Reply:
Yes, but why?

Reply:
As we discussed off-line, class with privates are real nominal types in TS, where a unique symbol, even not directly exported, leaks in the types through `keyof DefineKindHandle`